Docs updates for usage stats (#3393)
FusionAuth/fusionauth-site: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

### Collected Metrics

Currently FusionAuth sends a payload to the metrics endpoint for all instances. A self-hosted community user can disable network egress or block access to the metrics endpoint without affecting functionality. Customers on a paid plan must allow a metrics connection [per the license agreement](/license), unless they have an air-gapped license. 

> If you would like to learn more about what metrics we collect and how they are used, you can find that information [here](/docs/get-started/download-and-install/collected-metrics).

Reviewer: ```suggestion
To learn more about what metrics we collect and how they are used, view the [collected metrics documentation](/docs/get-started/download-and-install/collected-metrics).
```
